public/api/conf/1.0/testdata/get-pensions-dashboard.md

```diff
@@ -0,0 +1,178 @@
+<table>
+    <thead>
+        <tr>
+            <th>Scenario</th>
+            <th>NINO</th>
+            <th>Response</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <th colspan="3">More than 10 qualifying years</th>
+        </tr>
+        <tr>
+            <th colspan="3">Forecast</th>
+        </tr>
+        <tr>
+            <td><p>Taxpayer who is predicted to receive the full rate</p></td>
+            <td><p><code>LG000001A</code></p></td>
+            <td><p>200 (OK) Payload as regular example above</p></td>
+        </tr>
+        <tr>
+            <td><p>Taxpayer who is predicted to receive less than the full rate</p></td>
+            <td><p><code>LG001101A</code></p></td>
+            <td><p>200 (OK) Payload as regular example above</p></td>
+        </tr>
+        <tr>
+            <th colspan="3">Personal maximum</th>
+        </tr>
+        <tr>
+            <td><p>Taxpayer who is predicted to receive the full rate by paying gaps</p></td>
+            <td><p><code>LG000601A</code></p></td>
```

Comment:
I believe you'll be okay just using <td><code> LG000601A</code></td>

Reply:
just remove the p tag and just have the code tag, github formatted the tags i've done

Reply:
remove the p tag on all instances within the file?

Reply:
just where the <code> tags are used for the nino's

Reply:
p tags removed around NINOs in this file and get-state-pension.md (where I cloned data for the new dashboard.md file)